sdks/python/apache_beam/transforms/util.py

```diff
@@ -802,6 +802,20 @@ class BatchElements(PTransform):
   corresponding to its contents. Each batch is emitted with a timestamp at
   the end of their window.
 
+  When the max_batch_duration_secs arg is provided, a stateful implementation
+  of BatchElements is used to batch elements across bundles. This is most
+  impactful in streaming applications where many bundles only contain one
+  element. Larger max_batch_duration_secs values `might` reduce the throughput
```

Comment:
```suggestion
  element. Larger max_batch_duration_secs values might reduce the throughput
```

apologies, if i added the backticks here, they were unintentional